static/src/components/AnnotationButton.vue

```diff
@@ -43,7 +43,6 @@ export default defineComponent({
         await API.saveAnnotation(this.taskId, this.imageId, {
           label: this.label,
           hash: hash,
-          username: this.username,
```

Comment:
Can we remove the username prop now too?

Reply:
Yeah, but I'd suggest moving this to the other front-end related username/fullname PR - What do you think?